src/ui/MessageContent/index.tsx

```diff
@@ -266,7 +266,7 @@ export default function MessageContent({
               message={message as UserMessage | FileMessage}
               userId={userId}
               isByMe={isByMe}
-              isUnavailable={(replyType === 'THREAD' && (channel?.joinedAt * 1000) > message?.parentMessage?.createdAt)}
+              isUnavailable={(replyType === 'THREAD' && (channel.joinedAt * 1000) > (message.parentMessage?.createdAt ?? 0))}
```

Comment:
`message.parentMessage ` cc @HoonBaek

Reply:
@HoonBaek  Its type looked okay without the optional chaining, but let me know if there's any concern :)

Reply:
I'm worried about the case when the channel is null, how can we manage the case